flow/errors.py

```diff
@@ -61,3 +61,9 @@ def parse(self, parser):
     def err(self, msg, caller):
         """Raise a template error."""
         raise jinja2.TemplateError(msg)
+
+
+class IncompatibleSchemaVersion(RuntimeError):
+    """The FlowProject's configuration schema version is incompatible with this version of signac-flow."""  # noqa: E501
```

Comment:
Shorter docstring, to align with other error docstrings.
```suggestion
    """Indicates FlowProject schema is not supported by this signac-flow version."""
```